outputs.tf

```diff
@@ -49,6 +49,11 @@ output "aks_id" {
   value       = azurerm_kubernetes_cluster.main.id
 }
 
+output "aks_name" {
+  description = "The `aurerm_kubernetes-cluster`'s name"
```

Comment:
Can we add a dot in the description and readme file? Thanks!

```hcl
description = "The `aurerm_kubernetes-cluster`'s name."
```

Reply:
Added.